app/views/content/life-as-a-teacher/pay-and-benefits/teachers-pension-scheme.md

```diff
@@ -33,7 +27,7 @@ Once you join the scheme, it's quick and easy to manage your pension online.
 
 The teachers' pension scheme includes an employer contribution of 28.6%.
 
-It’s a defined benefit pension, which means it’s based on your salary rather than the amount you contribute.
+It’s a defined benefit pension, which means it’s based on your salary rather than the amount you contribute. Find out more about [teacher pay](/life-as-a-teacher/pay-and-benefits/teacher-pay).
```

Comment:
```suggestion
It’s a defined benefit pension, which means it’s based on your salary rather than the amount you contribute. [Find out more about teacher pay](/life-as-a-teacher/pay-and-benefits/teacher-pay).
```
Worth adding more context re destination for screen reader users?